ProxyAppsSuite/tests/miniamr.yaml

```
@@ -0,0 +1,904 @@
+               files: '{{test_runs.id}}.out'
+           meshTime_totalLoadBal_min:
+               regex: 'total\s+load\s+balance\s+\:\s+\S+\s\S+\s(\S+)\s\S+\s+'
+               files: '{{test_runs.id}}.out'
+           meshTime_totalLoadBal_max:
+               regex: 'total\s+load\s+balance\s+\:\s+\S+\s\S+\s\S+\s(\S+)\s+'
+               files: '{{test_runs.id}}.out'
+           num_plot_steps:
+               regex: 'Number\s+of\s+plot\s+steps\:\s+(\d+)\s*'
+               files: '{{test_runs.id}}.out' 
+           final_ranks:
+               regex: 'Summary\:\s+ranks\s+(\d+)\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           time_steps:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+(\d+)\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           total_time:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+(\S+)\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           calc:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+(\S+)\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           max_comm:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+(\S+)\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           min_red:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+(\S+)\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           total_refine:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+(\S+)\s+blocks\/ts\s+\S+\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           blocks_per_ts:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+(\S+)\s+max_blocks\s+\d+\s*'
+               files: '{{test_runs.id}}.out'
+           final_max_blocks:
+               regex: 'Summary\:\s+ranks\s+\d+\s+ts\s+\d+\s+time\s+\S+\s+calc\s+\S+\s+max\s+comm\s+\S+\s+min\s+red\s+\S+\s+refine\s+\S+\s+blocks\/ts\s+\S+\s+max_blocks\s+(\d+)\s*'
+               files: '{{test_runs.id}}.out' 
+   
+objects_1:
+   summary: 'runs the miniAMR test when num_objects = 1'
+   inherits_from: _base
+   slurm:
+     num_nodes: 7
+   variables:
+         test_runs:
```

```suggestion
         test_input:
```
Not totally necessary, it's just there's a module called `test_run` in the Pavilion source code.